pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -70,7 +72,9 @@ stdenv.mkDerivation {
   # so that `SPARSE_FLAGS' corresponds to the current architecture...
   #doCheck = true;
 
-  installFlags = "NO_INSTALL_HARDLINKS=1";
+  installFlags = "NO_INSTALL_HARDLINKS=1"
+    + (if withpcre2 then " USE_LIBPCRE2=1" else "");
```

Comment:
@vcunat only libpcre2 is recommended by git https://github.com/git/git/blob/master/Documentation/RelNotes/2.14.0.txt#L29
@Mic92 I think you're right and I've updated the pull request to disable pcre2 by default in gitMinimal